test/shared/avatarable_shared_tests.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AvatarableSharedTests
+  def self.included spec
```

Comment:
I thought this was RSpec for a minute until I realised it's using the shoulda gem!